packages/mask/src/plugins/Trader/constants/sushiswap.ts

```diff
@@ -35,4 +35,5 @@ export const SUSHISWAP_BASE_AGAINST_TOKENS: ERC20AgainstToken = {
     [ChainId.xDai]: [WNATIVE, USDC, USDT, WBTC].map((x) => x[ChainId.xDai]),
     [ChainId.Celo]: [WNATIVE, CUSD, CEUR].map((x) => x[ChainId.Celo]),
     [ChainId.Fantom]: [WNATIVE, DAI, USDC, fUSDT, WBTC].map((x) => x[ChainId.Fantom]),
+    [ChainId.Avalanche]: [WNATIVE, DAI, USDC, fUSDT, WBTC].map((x) => x[ChainId.Avalanche]),
```

Comment:
Please use the correct tokens here. You might need to find them from the sushiswap codebase.